projects/storefrontlib/src/cms-components/misc/global-message/global-message.component.html

```diff
@@ -19,8 +19,27 @@
     </button>
   </div>
   <div
-    class="alert alert-warning"
+    class="alert alert-info"
```

Comment:
This is a bug fix, but it's a breaking change at the same time. We have 2 options:
a) guard it with a featureLevel directive OR
b) target this PR directly to the branch `release/2.0.x`
We are close to the release of 2.0, so I recommend option b.

Why it's a breaking change:
- changing a CSS class is a DOM change (some customers may rely on this class, i.e. in their styling).
- it changes the behavior of code (some customers may rely on that dispatching a global message with type "information" shows a yellow alert).

See docs for maintaining public API: https://sap.github.io/cloud-commerce-spartacus-storefront-docs/breaking-changes/#avoiding-breaking-changes

Reply:
changed PR base to release/2.0.x